ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/models/Type.java

```diff
@@ -25,13 +25,14 @@
 import org.slf4j.LoggerFactory;
 
 import java.text.ParseException;
+import java.util.Set;
 
 public class Type {
```

Comment:
I have the same comment on this class that I do for Element.  Too many responsibilities.